tests/robustness/scenarios.go

```diff
@@ -65,10 +66,10 @@ func scenarios(t *testing.T) []testScenario {
 	}
 	enableLazyFS := e2e.BinPath.LazyFSAvailable()
 	baseOptions := []e2e.EPClusterOption{
-		e2e.WithSnapshotCount(100),
 		e2e.WithGoFailEnabled(true),
-		e2e.WithCompactionBatchLimit(100),
-		e2e.WithWatchProcessNotifyInterval(100 * time.Millisecond),
+		options.WithSnapshotCount(100),
+		options.WithExperimentalCompactionBatchLimit(100),
+		options.WithExperimentalWatchProgressNotifyInterval(100 * time.Millisecond),
```

Comment:
This PR adds a lot of util code, but at the end doesn't implement any randomizing. Working in isolation can lead to design straying away from the goal. 

Could we start from making 1 or 2 parameters random before we add 250 lines of util functions and tests?